src/other/Table.js

```diff
@@ -469,18 +503,22 @@
             }
             this.fixedCol
                 .style("position", "absolute")
-                .style("margin-top", -this.tableDiv.node().scrollTop + tableMarginHeight + "px")
+                .style("margin-top", -this.tableDiv.node().scrollTop + parseInt(tableMarginHeight) + "px")
             ;
             fixedColTd
                 .style("width", fixedColWidth + "px")
             ;
             this.fixedColHead
                 .style("position", "absolute")
-                .style("margin-top", (this.fixedHeader() ? this.tableDiv.node().scrollTop: 0) - tableMarginHeight + "px")
+                .style("margin-top", (this.fixedHeader() ? this.tableDiv.node().scrollTop: 0) - parseInt(tableMarginHeight) + "px")
```

Comment:
I would move the parseInt up to where the value was originally assigned (if possible) - then this instance would not have been needed....